test/Mono.Linker.Tests.Cases/Attributes/CoreLibraryAssemblyAttributesAreKept.cs

```diff
@@ -7,6 +7,9 @@ namespace Mono.Linker.Tests.Cases.Attributes
 {
 	[Reference ("System.dll")]
 	[SetupLinkerCoreAction ("link")]
+	// IComponent has TypeConverterAttribute referencing System.dll, which has
+	// unresolved references to Win32 assemblies.
+	[SetupLinkerArgument ("--skip-unresolved", "true")]
```

Comment:
I don't understand this. Is ths PR is making the situation worse instead of better?